README.md

```diff
@@ -62,6 +62,12 @@ func main() {
     for _, domain := range domainsResponse.Data {
         fmt.Println(domain)
     }
+
+    // List methods support a variety of options to paginate, sort and filter records.
+    // Here's a few example:
+
+    // get the list of domains filtered by name and sorted by expiration
+    client.Domains.ListDomains(whoami.Account.ID, &DomainListOptions{NameLike: "com", Sort: "expiration:DESC"})
```

Comment:
@weppos This should be `desc` instead. ..or we can add case insensitive support on the server side.